app/src/auto_validator/core/utils/ssh.py

```diff
@@ -66,6 +63,10 @@ def copy_files_to_remote(self, local_files: str, remote_path: str) -> None:
         if self.client.get_transport() is None or not self.client.get_transport().is_active():
             raise Exception("SSH connection is not open")
 
+        # Ensure local_files is a list
+        if isinstance(local_files, str):
+            local_files = [local_files]
+
```

Comment:
```suggestion
```
don't try to fix input, error out instead. Static code analyzer (or IDE) should be used to check types, if we really need it (not in this project though)